File: Level_1/문자열 다루기 기본/String_sn.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        int len = s.length();
        
        if(len == 4 || len == 6){
            for(int i = 0; i < len; i++){
                if(!Character.isDigit(s.charAt(i))){
                    answer = false;
                    break;
                }
            }
        }else answer = false;
       
        
        return answer;
    }
}

File: Level_1/문자열 다루기 기본/String_sn.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <ctype.h>

bool solution(const char* s) {
    bool answer = true;
    if(strlen(s) == 4 || strlen(s) == 6){
        for(int i = 0; i < strlen(s); i++){
            if(!isdigit(s[i])) return false;
        }
    }else return false;
    
    return answer;
}
